src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/data.html.twig

```diff
@@ -21,6 +21,10 @@
  * @author    PrestaShop SA and Contributors <contact@prestashop.com>
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
- *#}
+ * #}
 
-{{ record[column.options.field] }}
+{% if column.options.max_displayed_characters > 0 %}
```

Comment:
I just tested it and the ellipsis is always shown I think a condition is missing here

Reply:
```suggestion
{% if column.options.max_displayed_characters > 0 and column.options.field|length > column.options.max_displayed_characters %}
```

Reply:
Not sure about `|length` filter, but something similar ^^